workflows-scripts/data-export/export-marc-bib-records/dataExport_exportMARCBibRecords.jmx

```diff
@@ -551,14 +551,14 @@
               <hashTree/>
             </hashTree>
             <ConstantTimer guiclass="ConstantTimerGui" testclass="ConstantTimer" testname="Constant Timer" enabled="true">
-              <stringProp name="TestPlan.comments">Add constant timer of 5 seconds to mimics UI behavior</stringProp>
-              <stringProp name="ConstantTimer.delay">5000</stringProp>
+              <stringProp name="TestPlan.comments">Add constant timer of 3 seconds to mimics UI behavior</stringProp>
+              <stringProp name="ConstantTimer.delay">3000</stringProp>
```

Comment:
Isn't it still 5 seconds?

Reply:
As per the FOLIO UI chrome developer console, polling happens every 3 seconds now in Honeysuckle.